contracts/src/BattleShip.sol

```diff
@@ -5,13 +5,22 @@ import "./interfaces/IGame.sol";
 
 contract BattleShip is IGame {
     uint256 public number;
+    bytes32 public digest;
+
+    function initializeGame(uint256 gameId) external {
```

Comment:
* this needs `secretHash` or `secretGameStateHash` field, and then it goes to the event as well
* can we make it shorter? Like `initGame`